config/owasp/suppressions.xml

```diff
@@ -11,8 +11,8 @@
       <notes><![CDATA[name: io.netty 4.1.100.Final]]></notes>
       <cve>CVE-2023-4586</cve>
   </suppress>
-  <suppress until="2024-04-01">
-    <notes><![CDATA[name: azure-core-http-netty-1.13.10.jar]]></notes>
+  <suppress until="2024-08-01">
+    <notes><![CDATA[name: azure-core-http-netty-1.14.1.jar]]></notes>
```

Comment:
Do you still need to suppress it?

Reply:
yes, checked it without suppression , still it is not fixed